cves/CVE-2011-2204.yml

```diff
@@ -59,15 +67,15 @@ fixes_vcc_instructions: |
   was discovered in the "note" field.
   Refer to our instructions on how to find a Git SHA from an SVN revision.
 fixes:
-   - commit: 8b81c8c869987e35deed04993ecfcf7be27ca298
-     note: SVN rev 1140072, from the Tomcat website. This is in the Tomcat55 repo
-   - commit: 763a56b45999653ce648a18462b8a826809215b1
-     note: SVN rev 1140072, from the Tomcat website. This is in the standard tomcat (not Tomcat80, not Tomcat55) repo
+  - commit: 8b81c8c869987e35deed04993ecfcf7be27ca298
+    note: SVN rev 1140072, from the Tomcat website. This is in the Tomcat55 repo
+  - commit: 763a56b45999653ce648a18462b8a826809215b1
```

Comment:
Why isn't `eae54419c6` the VCC?

Reply:
I got these Git SHAs from the git commits themselves by going through the git logs. There's also the added challenge that they migrated from SVN to GIt in between, so I had to go through the revisions first. Where is `eae54419c6` coming from?